formwidgets/menuitems/assets/js/menu-items-editor.js

```diff
@@ -91,7 +91,13 @@
                 // If the saved title is the default new item title, use reference title,
                 // removing CMS page [base file name] suffix
                 if (selectedTitle && self.properties.title === self.$popupForm.attr('data-new-item-title')) {
-                    $titleField.val($.trim(selectedTitle.replace(/\s*\[.*\]$/, '')))
+                    var title = $.trim(selectedTitle.replace(/\s*\[.*\]$/, ''))
+                    var defaultLocale = $('[data-control="multilingual"]').data('default-locale')
+                    $titleField.val(title)
+                    if (defaultLocale) {
+                        var $localeField = $('[name="RLTranslate['+defaultLocale+'][title]"]', self.$popupContainer)
+                        $localeField.val(title)
+                    }
```

Comment:
@mjauvin can you explain what's going on here? I don't quite follow

Reply:
@LukeTowers when a reference page is selected, it sets the menu item title from the referenced page title.

this extra code also assigns the title to the RainLab.Translate default locale data locker